source/StigData/Processed/IISServer-10.0-2.7.xml

```diff
@@ -1533,7 +1532,7 @@ Double-click the "Error Pages" icon.
 
 Click any error message, and then click "Edit Feature Setting" from the "Actions" Pane. This will apply to all error messages.
 
-If the feature setting is not set to “Detailed errors for local requests and custom error pages for remote requests”, this is a finding.</RawString>
+If the feature setting is not set to "Detailed errors for local requests and custom error pages for remote requests", or "Custom error pages"  this is a finding.</RawString>
```

Comment:
es for remote requests", or "Custom error pages" - this may require a parser update to

Reply:
Disa changed the “ to "

This was parsed correctly.